src/main/python/main/ayab/hw_test.py

```diff
@@ -43,7 +43,19 @@
 class HardwareTestDialog(QDialog):
     """Console for hardware tests."""
 
-    commands = ["help", "send", "beep", "read", "auto", "test", "quit"]
+    commands = [
+        "help",
+        "send",
+        "beep",
+        # "setSingle",     # called when checking solenoids
+        # "setAll",        # no UI to handle argument
```

Comment:
why are these commented out

Reply:
I initially copied them all from the `Token` definition. Most test commands in the API get a button in the UI, except for these few which either can't really be called by the current UI (`setAll`) or are called using an alternative UI which isn't a simple button (`setSingle`). The intent of my comments was to explain why they were commented out, but I admit it's not very clear.

What do you think, should I just delete those lines?

Reply:
yeah I'd delete them

Reply:
done.